serverless.yml

```diff
@@ -25,5 +25,9 @@ functions:
     handler: le-aws-cloudwatch/le_cloudwatch.lambda_handler
     memorySize: 128
     timeout: 30
+    tags:
+      Service: Lambda
+      Project: Bill
```

Comment:
Can we pass the `Project` parameter value in as a variable to keep this project generic?

Reply:
Updated it 👍